settings/settings.json

```
@@ -234,6 +234,13 @@
                         "type": "integer",
                         "default": "365"
                     },
+                    {
+                        "title": "Enable Rewatching Next Up",
```

Can we change the wording to match the other next up setting above it? Renaming this would also make the list alphabetical again.

```suggestion
                        "title": "Rewatching Next Up",
```